pandas/core/strings.py

```diff
@@ -1651,17 +1651,34 @@ def str_translate(arr, table, deletechars=None):
 
 def str_get(arr, i):
     """
+    Extract element for each component.
+
     Extract element from lists, tuples, or strings in each element in the
     Series/Index.
 
     Parameters
     ----------
     i : int
-        Integer index (location)
+        Integer index (location).
```

Comment:
I find this not very clear. May be something like "Position of the element to extract"?

Reply:
Okay. Would make this change.